### PR TITLE
[DO-1687] Upgrade report for 'in progress' updates

### PR DESCRIPTION
**Pull request by Iva-khar**
Add status for 'in progress' register updates

### CODE REVIEW

File: data_ocean/converter.py [hunk 1]
```diff
@@ -244,6 +244,7 @@ def process(self, start_index=0):
                     traceback.print_exc()
                     print(msg)
                     raise Exception('Error!', msg)
+                    return False
```
Comment (roman-tiukh) [inline data_ocean/converter.py:247]: you cannot `return` after `raise`...

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: data_ocean/converter.py (hunk 1 above)
Comment (roman-tiukh) [reply data_ocean/converter.py:247]: are you tested this changes?

Comment (Iva-khar) [reply data_ocean/converter.py]: Not for exept. I remove `raise`, it needn't now.

**Review by roman-tiukh — CHANGES_REQUESTED** [review]
@IegorSoboliev please review this PR

File: business_register/converter/pep.py [hunk 2]
```diff
@@ -746,43 +747,46 @@ def update(self):
...
-            self.report.update_message += (f'Total records of RelatedPersonsLink from source is '
-                                           f'{converter.peps_links_total_records_from_source} '
-                                           f'but in our DB it is {peps_links_total_records}. \n')
-        peps_companies_total_records = CompanyLinkWithPep.objects.all().count()
-        if peps_companies_total_records != converter.peps_companies_total_records_from_source:
-            self.report.update_status = False
-            self.report.update_message += (f'Total records of CompanyLinkWithPep from source is '
-                                           f'{converter.peps_companies_total_records_from_source} '
-                                           f'but in our DB it is {peps_companies_total_records}')
+        if converter.process():
```
[the remark this answers is not in the document]
Comment (IegorSoboliev) [reply business_register/converter/pep.py:750]: I don`t think we need changes in this converter at all

[the remark this answers is not in the document]
Comment (Iva-khar) [reply business_register/converter/pep.py]: Removed all changes in pep.py

File: business_register/converter/pep.py [hunk 3]
```diff
@@ -746,43 +747,46 @@ def update(self):
...
+            self.report.update_finish = timezone.now()
             self.report.save()
```
[the remark this answers is not in the document]
Comment (Iva-khar) [reply business_register/converter/pep.py]: The same.

File: business_register/converter/pep.py [hunk 4]
```diff
@@ -746,43 +747,46 @@ def update(self):
...
+            self.measure_changes('business_register', 'Pep')
+            self.report.save()
+            logger.info(f'{self.reg_name}: Report created successfully.')
 
-        if self.report.update_status:
-            logger.info(f'{self.reg_name}: Update finished unsuccessfully')
+            if self.report.update_status:
+                logger.info(f'{self.reg_name}: Update finished unsuccessfully')
+            else:
+                logger.info(f'{self.reg_name}: Update finished unsuccessfully. See update_message')
```
[the remark this answers is not in the document]
Comment (Iva-khar) [reply business_register/converter/pep.py]: The same.

File: business_register/converter/pep.py [hunk 5]
```diff
@@ -746,43 +747,46 @@ def update(self):
...
+            peps_companies_total_records = CompanyLinkWithPep.objects.all().count()
+            if peps_companies_total_records != converter.peps_companies_total_records_from_source:
+                self.report.update_status = False
+                self.report.update_message += (f'Total records of CompanyLinkWithPep from source is '
+                                               f'{converter.peps_companies_total_records_from_source} '
+                                               f'but in our DB it is {peps_companies_total_records}')
+                self.report.save()
+
+            self.update_register_field(settings.PEP_REGISTER_LIST, 'total_records', peps_total_records)
+            self.report.save()
```
[the remark this answers is not in the document]
Comment (Iva-khar) [reply business_register/converter/pep.py]: The same.

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**